src/components/ReportCard/ReportCard.js

```diff
@@ -5,11 +5,9 @@ import PropTypes from 'prop-types'
 const ReportCard = ({ studentTeacherRatio, perStudentExpenditure, teacherSalaryInfo, studentPopulationSize, numberOfSchoolsInDistrict }) => {
     return (
         <div className='report-card'>
```

Comment:
you read my mind!!  I was thinking we were going to need to wrap these in a <p> tag!  nice